lib/shared/store.js

```diff
@@ -212,9 +213,12 @@ const storeReducer = (state = DEFAULT_STATE, action) => {
         })
       }
 
-      if (_.isNil(action.data.speed)) {
+      if (_.some([
+        (!_.isNumber(action.data.speed) || !_.isFinite(action.data.speed)) && action.data.speed !== null,
```

Comment:
And IMHO this should be rearranged to:
```
action.data.speed !== null && (!_.isNumber(action.data.speed) || !_.isFinite(action.data.speed))
```
because if `action.data.speed` _is_ null, then there's no point checking `_.isNumber` or `_.isFinite` ;-)

Reply:
Fixed these two now